ProcessMaker/pm-blocks
# FOUR-12515: Remove Script Executors References from Default PM Blocks

## Description

As part of our ongoing effort to address issues with PM Block Script Executors overwriting custom script executors during nightly sync, a refined approach is needed. This task involves both restoring PM Blocks to the main branch and removing script executor references from the Default PM Blocks JSON.

## Solution:
- Restore PM Blocks on the Main Branch
- Remove Script Executor References from PM Blocks:
- Identify and review PM Blocks containing references to script executors.
- Edit the corresponding JSON files to eliminate script executor references.
- Thoroughly test the application to verify the functionality of PM Blocks.
- Confirm that PM Blocks are free of script executor references and operate as expected.

## How to test 

- In the .env file of core, please update the PM_BLOCKS_BRANCH to:
-- `DEFAULT_PM_BLOCKS_BRANCH=task/FOUR-12515`
- Make a change to the PHP Executor script executor
- Run `php artisan package-pm-blocks:sync`

Please ensure that the PHP Executor remains unchanged.

## Related Tickets & Packages
Ticket: [FOUR-12515](https://processmaker.atlassian.net/browse/FOUR-12515)


## Code Review Checklist


[FOUR-11846]: https://processmaker.atlassian.net/browse/FOUR-11846?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ


[FOUR-12515]: https://processmaker.atlassian.net/browse/FOUR-12515?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
